File: DesignPatternsExercises/java/MementoDesignPattern/Exercise/LedTV.java
public class LedTV {
    private final int size;
    private final double price;

    public LedTV(int size, double price) {
        this.size = size;
        this.price = price;
    }

    @Override
    public String toString() {
        return "LedTV:\n        size: " + size + ", price: $" + price;
    }
}

File: DesignPatternsExercises/ruby/memento_design_pattern/solution/led_tv.rb
class LedTV
  def initialize(size, price)
    # __________(6)__________ = size
    @led_tv = size
    # __________(7)__________ = price
    @speaker = price
  end

  def to_string
    "Led TV:\n        size: #{@led_tv}, price: $#{@speaker}"
  end
end
